content/applications/websites/website/pages/seo.rst

```diff
@@ -33,7 +33,6 @@ finetune them. Make sure they fit the content of the page, otherwise you will
 be downgraded by search engines.
 
 .. image:: seo/seo01.png
```

Comment:
Images are missing the 'alt' description, a little sentence about what is being shown in the screenshot